orocos_kdl/src/CMakeLists.txt

```diff
@@ -128,6 +131,8 @@ ENDIF()
 
 # Needed so that the generated config.h can be used
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
can this line be replace by modern cmake?

Reply:
I think that's what line 134 is doing?  Maybe this can just be dropped.

Reply:
Correct, removed in 3efaaf9c97e6ec74cda2bd72ad52382fb9f373d0.